packages/react-components/react-skeleton/src/components/SkeletonItem/useSkeletonItemStyles.ts

```diff
@@ -0,0 +1,195 @@
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import type { SkeletonItemSlots, SkeletonItemState } from './SkeletonItem.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+import { tokens } from '@fluentui/react-theme';
+
+export const skeletonItemClassNames: SlotClassNames<SkeletonItemSlots> = {
+  root: 'fui-SkeletonItem',
+};
+
+const skeletonWaveAnimationRTL = {
+  from: {
+    backgroundPositionX: '300% /* @noflip */',
+  },
+  to: {
+    backgroundPositionX: '0% /* @noflip */',
+  },
+};
+
+const skeletonWaveAnimation = {
+  from: {
+    backgroundPositionX: '0% /* @noflip */',
+  },
+  to: {
+    backgroundPositionX: '300% /* @noflip */',
+  },
+};
+
+const skeletonPulseAnimation = {
+  from: {
+    opacity: '1',
+  },
+  to: {
+    opacity: '0.4',
+  },
+};
+
+/**
+ * Styles for the root slot
+ */
+const useStyles = makeStyles({
+  root: {
+    position: 'relative',
+    ...shorthands.overflow('hidden'),
+    backgroundSize: '300% 100%',
+    backgroundPositionX: 'center',
+    backgroundPositionY: 'center',
+    backgroundAttachment: 'fixed',
+    animationIterationCount: 'infinite',
+    animationDuration: '3s',
+    animationTimingFunction: 'linear',
+  },
+  wave: {
+    animationName: skeletonWaveAnimation,
+    backgroundImage: `linear-gradient(
+      to right,
+      ${tokens.colorNeutralStencil1} 0%,
+      ${tokens.colorNeutralStencil2} 50%,
+      ${tokens.colorNeutralStencil1} 100%)`,
+    '@media screen and (forced-colors: active)': {
+      backgroundColor: `WindowText
+      linear-gradient(
+        to right,
+        transparent 0%,
+        Window 50%,
+        transparent 100%)
+      `,
+    },
+  },
+  waveRtl: {
+    animationName: skeletonWaveAnimationRTL,
+    backgroundImage: `linear-gradient(
+      to right,
+      ${tokens.colorNeutralStencil1} 0%,
+      ${tokens.colorNeutralStencil2} 50%,
+      ${tokens.colorNeutralStencil1} 100%)`,
+    '@media screen and (forced-colors: active)': {
+      backgroundColor: `WindowText
+      linear-gradient(
+        to right,
+        transparent 0%,
+        Window 50%,
+        transparent 100%)
+      `,
+    },
```

Comment:
These appear to be identical to the `wave` class right? You don't need to repeat them here since you're also applying the base `wave` class in rtl anyways:
```suggestion
```